docs/installation.md

```diff
@@ -0,0 +1,103 @@
+# Installation
+
+## Deploy the Gateway
+
+You can deploy NGINX Kubernetes Gateway on an existing Kubernetes 1.16+ cluster. The following instructions walk through the steps for deploying on a [kind](https://kind.sigs.k8s.io/) cluster.
+
+1. Load the NGINX Kubernetes Gateway image onto your kind cluster:
+
+   ```
+   kind load docker-image nginx-kubernetes-gateway:0.0.1
```

Comment:
Or is the statement below meant to account for this? If so I'd still like to move this to "version" or "x.y.z" or something different.

Reply:
We can use the `edge` tag for main branch and the release specific tag for the docs in the release branch.
The docs do not account for that because it is out of scope of the PR (the PR doesn't fix the docs).